.all-contributorsrc

```diff
@@ -569,6 +569,15 @@
       "contributions": [
         "bug"
       ]
+    },
+    {
+      "login": "cs-hall",
+      "name": "cs-hall",
```

Comment:
hey @cs-hall, let me know if I should add your real name to this in addition to your GitHub handle. You can see how this ends up looking in the README of the repo: https://github.com/datalad-handbook/book#contributors- :) 

Also, please feel free to open a PR to add your details to the zenodo.json file of this repository. You can add yourself with your name, affiliation (optional), and orcid (optional) at the second to last spot, right before "Michael Hanke": https://github.com/datalad-handbook/book/blob/main/.zenodo.json
You can also leave the relevant information in a comment here, then I can add it. The current PR adds you to the handbook README as a contributor, your name in the zenodo.json files will add you as a coauthor to the handbook frontpage and the regular zenodo releases. :)

Reply:
Hey @adswa, Thanks! I believe I've created the PR on the zenodo.json file. Feel free to add my name (Christopher S. Hall)!

Reply:
Nice, I have changed it. 

> I believe I've created the PR on the zenodo.json file

There is no PR I can act upon in this repo, but I found a commit in your fork:
https://github.com/cs-hall/book/commit/9d4f47d0029be7638b2c4176b139a4334d604d99 and created a PR from it https://github.com/datalad-handbook/book/pull/953 
Thanks!